viewing_party/party.py

```diff
@@ -1,23 +1,151 @@
 # ------------- WAVE 1 --------------------
 
+from enum import unique
+
+# This is a helper function used in some above functions
+def get_users_watched_movie_titles(user_data):

```

Comment:
This appears to be used primarily to generate a list of titles for use with `in` checks. Consider generating a set instead, which has a much more efficient `in` behavior.

Also, consider defining this function closer to where it's used.